src/toffy/json_utils.py

```diff
@@ -229,13 +234,23 @@ def check_fov_resolutions(bin_file_dir, run_name, save_path=None):
         mult = 400 / fov_micron_length
         pixels_adj = int(mult * fov_pixel_length)
 
-        # output values
-        print(f"{default_name} ({custom_name}): {pixels_adj} pixels x {400} microns")
+        # save values
         resolutions.append(pixels_adj)
 
     resolution_data = pd.DataFrame(
         {"fov": fov_names, "name": custom_names, "pixels / 400 microns": resolutions}
     )
+
+    if len(set(resolution_data["pixels / 400 microns"])) > 1:
+        # print resolutions that are in the minority
+        print("Resolutions are not consistent among all FOVs.")
+
+        val = Counter(resolutions).most_common(1)[0][0]
+        subset = resolution_data[resolution_data["pixels / 400 microns"] != val]
+        print(subset.to_string(index=False))
```

Comment:
How clearly does this print in the notebook? `to_string` converts this to raw string format, which isn't the best to look at. Not sure if Jupyter does anything here.

In either case, personally I'd opt for more clarity, something like:

```
Resolutions are not consistent among all FOVs:

10: fov-1-scan-1, fov-3-scan-1, fov-4-scan-1
20: fov-2-scan-1, fov-5-scan-1
...
```

Reply:
I changed the output slightly for more clarity while only outputting fovs with inconsistent resolutions. `to_string` works well when the index is excluded!

<img width="499" alt="Screenshot 2023-08-21 at 2 43 30 PM" src="https://github.com/angelolab/toffy/assets/38049893/0448d7e3-b3e5-4fa1-928e-3fe1d141fbc9">